nexus/resource_blobstore.go

```diff
@@ -81,11 +81,29 @@ func resourceBlobstore() *schema.Resource {
 				Required:    true,
 			},
 			"path": {
-				ConflictsWith: []string{"bucket_configuration"},
+				ConflictsWith: []string{"bucket_configuration","bucket_name"},
 				Description:   "The path to the blobstore contents. This can be an absolute path to anywhere on the system nxrm has access to or it can be a path relative to the sonatype-work directory",
 				Type:          schema.TypeString,
 				Optional:      true,
 			},
+			"bucket_name": {
+				ConflictsWith: []string{"bucket_configuration","path"},
+				Description:   "The name of the Google Cloud Storage bucket. Needed for blobstore type 'Google Cloud Storage'",
+				Type:          schema.TypeString,
+				Optional:      true,
```

Comment:
I think this should be non-optional

Reply:
Has to be optional as the type is shared. If it wasn't optional then you'll be stuck when configuring S3 or file